crates/bevy_core_widgets/src/core_button.rs

```diff
@@ -24,7 +26,7 @@ pub struct CoreButton {
     /// Optional system to run when the button is clicked, or when the Enter or Space key
     /// is pressed while the button is focused. If this field is `None`, the button will
     /// emit a `ButtonClicked` event when clicked.
-    pub on_click: Option<SystemId>,
+    pub on_click: Callback,
```

Comment:
How come all other `on_*` fields are `on_change` but specifically for button it is `on_click`?
Especially since it seems it will also be used for non-clicks, such as the described Enter or Space key while focused.

Reply:
`on_activate` sounds good 👍

Reply:
Done.